recipe/meta.yaml

```diff
@@ -28,11 +28,6 @@ build:
     - '**/libm.so.6'             # [linux]
     - '**/ld-linux-x86-64.so.2'  # [linux]
     - '**/ld64.so.1'             # [linux]
-  ignore_run_exports:
-    - pixman
-    - freetype
-    - fontconfig
-    - gdk-pixbuf
 
 requirements:
   build:
```

Comment:
I found `bison` only in [CMakeLists.txt](https://gitlab.com/graphviz/graphviz/-/blob/main/CMakeLists.txt) but we use cmake only on win. Do we really need it on unix?

Reply:
yes, from my findings it seems to be used in some sub-code-paths